docs/api/js/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html><html class="default no-js"><head><meta charSet="utf-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><title>ONNX Runtime JavaScript API</title><meta name="description" content="Documentation for ONNX Runtime JavaScript API"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link rel="stylesheet" href="assets/style.css"/><link rel="stylesheet" href="assets/highlight.css"/><script async src="assets/search.js" id="search-script"></script></head><body><header><div class="tsd-page-toolbar"><div class="container"><div class="table-wrap"><div class="table-cell" id="tsd-search" data-base="."><div class="field"><label for="tsd-search-field" class="tsd-widget search no-caption">Search</label><input type="text" id="tsd-search-field"/></div><ul class="results"><li class="state loading">Preparing search index...</li><li class="state failure">The search index is not available</li></ul><a href="index.html" class="title">ONNX Runtime JavaScript API</a></div><div class="table-cell" id="tsd-widgets"><div id="tsd-filter"><a href="#" class="tsd-widget options no-caption" data-toggle="options">Options</a><div class="tsd-filter-group"><div class="tsd-select" id="tsd-filter-visibility"><span class="tsd-select-label">All</span><ul class="tsd-select-list"><li data-value="public">Public</li><li data-value="protected">Public/Protected</li><li data-value="private" class="selected">All</li></ul></div> <input type="checkbox" id="tsd-filter-inherited" checked/><label class="tsd-widget" for="tsd-filter-inherited">Inherited</label><input type="checkbox" id="tsd-filter-externals" checked/><label class="tsd-widget" for="tsd-filter-externals">Externals</label></div></div><a href="#" class="tsd-widget menu no-caption" data-toggle="menu">Menu</a></div></div></div></div><div class="tsd-page-title"><div class="container"><h1>ONNX Runtime JavaScript API</h1></div></div></header><div class="container container-main"><div class="row"><div class="col-8 col-content"><div class="tsd-panel tsd-typography">
+<a href="#onnx-runtime-javascript-api" id="onnx-runtime-javascript-api" style="color: inherit; text-decoration: none;">
+  <h1>ONNX Runtime JavaScript API</h1>
+</a>
+<p>ONNX Runtime JavaScript API is a unified API for all JavaScript usages. It&#39;s dependency of the following NPM packages:</p>
+<ul>
+<li>onnxruntime-node</li>
+<li>onnxruntime-web</li>
+<li>onnxruntime-react-native</li>
+</ul>
+<p>This package (onnxruntime-common) is not designed for using directly. Please consider to install one of the NPM packages above according to target platform.</p>
```

Comment:
"This package" <- it's not here, so we can directly use "onnxruntime-common is not designed ....".

Reply:
this section is directly generated from js/common/README.md. maybe I can just remove the README section, or write a summary for the API document

Reply:
This is updated, but need to wait for #9077 get merged and update the re-generated docs

Reply:
updated to latest